### PR TITLE
Update Readme and archtecture diagram.

### PR DESCRIPTION
**Pull request by scottdangelo**
Update the Readme for changes to naming of Bluemix.
Update the architecture diagram and add Flow steps.

### CODE REVIEW

File: ?
Comment (markstur) [inline ?:1]: link says bluemix but dest changed

Comment (markstur) [inline ?:1]: link says bluemix but dest changed

Comment (markstur) [inline ?:1]: looks like a search/replace error on this repo link

Comment (markstur) [inline ?:1]: you should remove the ones that aren't related.  No powerai, spark, k8s.  And above I'd remove Data Analytics Code Patterns.

Comment (markstur) [inline ?:1]: s/thier/their/

Comment (markstur) [inline ?:1]: s/applicsation/application/

**Review by markstur — CHANGES_REQUESTED** [review]
see comments

File: ?
Comment (markstur) [inline ?:1]: s/send/sends/

Comment (markstur) [inline ?:1]: caps don't work in links

Comment (markstur) [inline ?:1]: caps don't work in links

Comment (markstur) [inline ?:1]: 2 lines got combined here

**Review by markstur — DISMISSED** [review]
still some issues unless I looked at an old diff

**Review by stevemar — APPROVED** [review]
this looks good now

**Pull request merged**